src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -325,6 +325,10 @@ emit_simd_ins_for_binary_op (MonoCompile *cfg, MonoClass *klass, MonoMethodSigna
 			case SN_Divide:
 			case SN_op_Division: {
 				const char *class_name = m_class_get_name (klass);
+				if (!strcmp("Vector4", class_name) && fsig->params [1]->type == MONO_TYPE_R4) {
```

Comment:
This new condition seems to be redundant. The following condition is sufficient enough.

Reply:
Will modify it in the next PR